<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingTypeDialog</class>
 <widget class="QDialog" name="SettingTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标注类型配置</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">3</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>761</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>216</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>216</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNo"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>77</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdate">
     <property name="text">
      <string>更新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>颜色：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxColor"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
